app/src/main/python/gogdl/languages.py

```
@@ -0,0 +1,123 @@
+    Language("ms-MY", "Malay", "Bahasa Malaysia", []),
+    Language("mt-MT", "Maltese", "Malti", []),
+    Language("nb-NO", "Norwegian", "Norsk", ["no"]),
+    Language("nl-NL", "Dutch", "Nederlands", ["nl"]),
+    Language("ns-ZA", "Northern Sotho", "Sesotho sa Leboa", []),
+    Language("pa-IN", "Punjabi", "ਪੰਜਾਬੀ", []),
+    Language("pl-PL", "Polish", "Polski", ["pl"]),
+    Language("ps-AR", "Pashto", "پښتو", []),
+    Language("pt-BR", "Portuguese (Brazilian)", "Português do Brasil", ["br"]),
+    Language("pt-PT", "Portuguese", "Português", ["pt"]),
+    Language("ro-RO", "Romanian", "Română", ["ro"]),
+    Language("ru-RU", "Russian", "Pусский", ["ru"]),
```

Ruff (0.12.2)

String contains ambiguous `у` (CYRILLIC SMALL LETTER U). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF001)